src/components/PrivacyPolicy.vue

```diff
@@ -248,7 +248,7 @@
   <h3>4. How your personal data is protected and stored</h3>
 
   <p>
-    We implement physical, technical, and organizational measures designed to
+    We implement technical and organizational measures designed to
```

Comment:
Please fix (check the warning)

Reply:
What warning?

Reply:
https://github.com/UrbanSystemsLab/climateiq-login/pull/15/files

Check warning on line 251 in src/components/PrivacyPolicy.vue
GitHub Actions / build (20.x)
Replace...